korge/src/commonMain/kotlin/com/soywiz/korge/view/RectBase.kt

```diff
@@ -44,9 +44,9 @@ open class RectBase(
 	protected open val bheight get() = 0.0
 
     //@KorgeInternal
-    override val anchorDispX get() = (anchorX * bwidth)
+    override val anchorDispX get() = when { baseBitmap.trimmed -> (anchorX * baseBitmap.virtFrame.width - baseBitmap.virtFrame.x) else -> (anchorX * bwidth) }
```

Comment:
Since there are only two branches here, could we use an if instead to be consistent with the codebase?

Reply:
bwidth is not being used in the trimmed version branch, and seems to be `protected open val`. Don't remember / didn't check yet the implementors but we should be sure this is working properly in several cases

Reply:
Good idea. Could think of setting up AnchorDispX and Y from local properties or members. So we wouldn't need also an if clause. But instead set the properties depending if trimmed or not. I will look into it. I am currently on the road with the kids ;)